README.md

```diff
@@ -44,9 +48,20 @@ Please refer to our documentation at [OpenEBS Documentation](http://docs.openebs
 
 Head over to the [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-## Community
+## Community, discussion, and support
+
+Learn how to engage with the OpenEBS community on the [community page](https://github.com/openebs/openebs/tree/master/community).
+
+You can reach the maintainers of this project at:
+
+- [Kubernetes Slack](http://slack.k8s.io/) channels: 
+      * [#openebs](https://kubernetes.slack.com/messages/openebs/)
+      * [#openebs-dev](https://kubernetes.slack.com/messages/openebs-dev/)
+- [Mailing List](https://lists.cncf.io/g/cncf-openebs-users)
+
+### Code of conduct
 
-See the [OpenEBS Community page](https://github.com/openebs/openebs/tree/master/community) for reaching out to the OpenEBS Developers.
+Participation in the OpenEBS community is governed by the [CNCF Code of Conduct](CODE-OF-CONDUCT.md).
```

Comment:
question: Is this a new change that we need to take up in all other repos also? In other repos we have a CODE_OF_CONDUCT.md file present.

Reply:
yes. it should be like this. https://github.com/openebs/linux-utils/blob/master/CODE_OF_CONDUCT.md